db/db_impl/db_impl_write.cc

```diff
@@ -1147,6 +1147,7 @@ Status DBImpl::WriteToWAL(const WriteThread::WriteGroup& write_group,
   // Same holds for all in the batch group
   size_t write_with_wal = 0;
   WriteBatch* to_be_cached_state = nullptr;
+  StopWatch write_sw(env_, stats_, DB_WRITE_WAL_TIME);
```

Comment:
Let's add the same stopwatch to `ConcurrentWriteToWAL` as well.

Reply:
OK, have added.